chapters/appendix-IX-file-tags.md

```diff
@@ -0,0 +1,45 @@
+# Appendix IX: SPDX File Tags
+
+## Rationale
+
+SPDX short-form license identifiers, described in [Appendix V](./appendix-V-using-SPDX-short-identifiers-in-source-files.md), provide a mechanism for developers to easily convey information about the licenses they declare on a file-by-file basis. That mechanism is intentionally very easy for software tools to identify and detect, since it includes a standard text string (`SPDX-License-Identifier:`) that is unlikely to occur in other contexts, and since it uses license identifiers from the [SPDX License List](https://spdx.org/licenses) or from user-defined `LicenseRef-` statements.
+
+The SPDX specification defines various other fields in the [File Information](./4-file-information.md) section that are also useful for conveying information on a file-by-file basis. This appendix describes a mechanism, similar to `SPDX-License-Identifier`, for developers to convey such other file-based information easily in comments in their files. This in turn enables software tools to easily find and extract that information, and to insert it into the corresponding fields of an SPDX document generated by those tools.
+
+## Format
+
+An SPDX file tag consists of a single line, generally as part of a comment, in the following format:
+
+    SPDX-tagname: <value>
+
+where _tagname_ is replaced by the 'tag' defined for tag-value SPDX documents for that field, according to the [File Information](./4-file-information.md) section of the SPDX specification. The meaning and semantics of any SPDX file tag are intended to be identical to those described in the File Information section of the SPDX specification.
+
+Examples:
+
+File type (see section 4.3):
+
+    SPDX-FileType: SOURCE
+    SPDX-FileType: DOCUMENTATION
+    SPDX-FileType: TEXT
+
+Copyright text (see section 4.8):
+
+    SPDX-FileCopyrightText: 2019 Jane Doe <jane@example.com>
+    SPDX-FileCopyrightText: Copyright 2008-2010 John Smith
+    SPDX-FileCopyrightText: Copyright Example Company
+    SPDX-FileCopyrightText: Copyright contributors to the Foo project.
```

Comment:
These examples suggest that adding "Copyright" after the tag is standard practice. Is it, or should it be?

Reply:
From SPDX's perspective, we would not have a recommendation either way. Similar to how the FileCopyrightText part of the SPDX specification doesn't advise developers on how their copyright notices should, or should not, be formatted.

I understand that REUSE has some guidance on this (and that's great to see!) but for purposes of SPDX I believe we try to steer clear of anything that is arguably a legal judgment to make.

Reply:
FYI, here's where I mentioned this a while back in the original issue thread: https://github.com/spdx/spdx-spec/issues/122#issuecomment-492403213

Reply:
That's fine by me :)